### PR TITLE
REPO-3918 updates for java 11

### PR DESCRIPTION
**Pull request by mmuller88**

### CODE REVIEW

**Comment by derekhulley** [comment]
Changed the number of reviewers required to 1.
We do, however, need to look at the changes and see that they make sense before allowing changes in, even if they are just looking at what has been done as an approach.

File: pom.xml [hunk 1]
```diff
@@ -501,11 +506,10 @@
         <plugins>
             <plugin>
                 <artifactId>maven-site-plugin</artifactId>
-                <version>3.5.1</version>
```
Comment (mmuller88) [inline pom.xml:504]: Github seems not to render this correct. I updated it to 3.7.1

Comment (ancutam) [reply pom.xml:504]: There is an update to 3.7.1 upper, line 179 ...

Comment (mmuller88) [reply pom.xml:504]: ahh right I understand

**Comment by ancutam** [comment]
I'm thinking maybe create a test/rc version of the super pom (10-test), have that uploaded to Nexus, use it in the feature branches for a while, just to make sure all the changes are in, or the right dependencies versions.
And after that, we release (copy to maven central) the "official" 10 version.
Might be a bit overkill, but I can easily see this getting to version 11, 12 and complicate things.
cc: @derekhulley @skopf

**Comment by derekhulley** [comment]
*10-rc1* ... maybe.  It won't take much longer and may save the DelEng team some frustration.
But, why not just go through and do a 10 (in Nexus) and then however many more we need before we finally ask DelEng to do the copy.

**Comment by mmuller88** [comment]
I would go with 10-rc1 without pushing to maven central. And if we are happy we release 10 to maven central as well.

**Review by ancutam — APPROVED** [review]

**Pull request merged**